recipes/cms-fireworks/meta.yaml

```diff
@@ -0,0 +1,88 @@
+{% set name = "fwlite" %}
+{% set version = "10.5.0" %}
+{% set hash = "2f5c169b6ad54330f8c8ffee8ac287c719b579c2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/gartung/fwlite.git
+  git_rev: {{ hash }}
+  folder: src
+
+build:
+  number: 0
+  skip: True  # [win or py3k]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - cmake
+    - git
+    # Required for OpenGL support
+    # See https://conda-forge.org/docs/maintainer/knowledge_base.html?highlight=cdt#libgl
+    - {{ cdt('mesa-libgl-devel') }}  # [linux]
+    - {{ cdt('mesa-dri-drivers') }}  # [linux]
+    - {{ cdt('libselinux') }}  # [linux]
+    - {{ cdt('libxdamage') }}  # [linux]
+    - {{ cdt('libxxf86vm') }}  # [linux]
+  host:
+    - boost =1.68.0
+    - clhep =2.4.0.2
+    - cms-md5
+    - hepmc2
+    - libsigcpp
+    - libuuid
+    # - openssl
```

Comment:
This can be uncommented once ROOT is rebuilt (will happen later today).

Reply:
Actually this isn't going to happen today so I'm happy to merge once you've added the geometry root files. (I presume they should be a runtime dependency)

Reply:
I need to create an activate.sh script. I am also trying to determine why cmsShow.exe is recompiling the headers when started.